prody/proteins/interactions.py

```diff
@@ -0,0 +1,2043 @@
+                          i.distance, None, i[0].coords, i[2].coords]           
+             
+        if str(type(i)).split('.')[-1].strip("'>") == 'waterbridge':
+            water = i.water
+            Inter_list = ['waterbridge',i.restype+str(i.resnr), i[0].type+'_'+str(i[0].idx), i.reschain,
+                          i.restype+str(i.resnr_l), i[3].type+'_'+str(i[3].idx), i.reschain_l, 
+                          [i.distance_aw, i.distance_dw], [i.d_angle, i.w_angle], i[0].coords, i[3].coords, 
+                          i.water.coords, i[7].residue.name+'_'+str(i[7].residue.idx)]    
+        else: pass
+                      
+        Inter_list_all.append(Inter_list)               
+    
+    for nr_k,k in enumerate(Inter_list_all):
+        LOGGER.info("%3i%22s%10s%26s%4s  <---> %8s%12s%4s%6.1f" % (nr_k,k[0],k[1],k[2],k[3],k[4],k[5],k[6],k[7]))
+    
+    return Inter_list_all
+
+
+def showProteinInteractions_VMD(atoms, interactions, color='red',**kwargs):
+    """Save information about protein interactions to a TCL file (output)
+    which can be further use in VMD to display all intercations in a graphical interface
+    (in TKConsole: play script_name.tcl).
+    Different types of interactions can be saved separately (color can be selected) 
+    or all at once for all types of interactions (hydrogen bonds - blue, salt bridges - yellow,
+    pi stacking - green, cation-pi - orange and hydrophobic - silver).
+    
+    :arg atoms: an Atomic object from which residues are selected
+    :type atoms: :class:`.Atomic`
+    
+    :arg interactions: List of interactions for protein interactions.
+    :type interactions: List of lists
+    
+    :arg color: color to draw interactions in VMD,
+                not used only for single interaction type.
+    :type color: str or **None**, by default `red`.
+    
+    :arg output: name of TCL file where interactions will be saved.
+    :type output: str
+        
+    Example (hydrogen bonds for protein only): 
+    >> interactions = calcHydrogenBonds(atoms.protein, distA=3.2, angle=30)
+    or all interactions at once:
+    >> interactions = calcProteinInteractions(atoms.protein) """
+
+    try:
+        coords = (atoms._getCoords() if hasattr(atoms, '_getCoords') else
+                    atoms.getCoords())
+    except AttributeError:
+        try:
+            checkCoords(coords)
+        except TypeError:
+            raise TypeError('coords must be an object '
+                            'with `getCoords` method')
+
+    if not isinstance(interactions, list):
+        raise TypeError('interactions must be a list of interactions.')
+    
+    try:
+        output = kwargs['output']
+    except:
+        output = atoms.getTitle()+'_interaction.tcl'
+    
+    tcl_file = open(output, 'w') 
+    
+    
+    def TCLforSingleInteaction(interaction, color='blue', tcl_file=tcl_file):
+        """Protein interactions from ProDy for a single Interaction"""
```

Comment:
I know this is an internal function, but it could still be good to have better documentation of what this means

Reply:
Done